src/components/Accordion/Accordion.tsx

```diff
@@ -137,7 +137,7 @@ const AccordionContent = React.forwardRef<
   <RadixAccordion.Content
     ref={ref}
     className={cn(
-      'text-body-secondary px-md pt-md pb-xl overflow-hidden',
+      'text-body-secondary px-md pt-xs pb-md overflow-hidden',
```

Comment:
The original padding was following the Figma design. However, in combination with DataSheets, I believe the updated padding fixes it visually. This is because the Figma design does not use any padding for the DataSheet rows, which I believe should be used to improve the structure:

<img width="627" height="135" alt="Image" src="https://github.com/user-attachments/assets/b78dfa85-561e-4522-a0f5-7d44787600a8" />